<?xml version="1.0"?>
<launch>

    <param name="constraint_graph_path" value="/home/daniel/Downloads/export//FLOAM_Mon_2023.03.06_09:18:51/graph.pg" />
    <!--arg name="slam_param_file" default="$(find lio_sam)/config/params_structor_outdoor.yaml" /-->
    <param name="keyframe_min_transl" type="double" value="0.1" /> <!-- 0.05 -->
    <param name="keyframe_min_rot" type="double" value="50" /> <!-- 0.035 -->
    <param name="use_keyframe" type="bool" value="true" />
    <param name="inner_iterations" type="double" value="2" />
    <param name="outer_iterations" type="double" value="2" />
    <param name="max_dist_association" type="double" value="1.0" />
    <param name="tot_scans" type="double" value="3" />



    <node pkg="dmapping_refinement" type="refinement_node" name="refinement_node" output="screen"/>
    
    <arg name="rviz" default="true" />
    <group if="$(arg rviz)">
        <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find dmapping_refinement)/rviz/slam.rviz" />
    </group>
    
    

</launch>

